buildx/working-with-buildx.md

```diff
@@ -16,13 +16,7 @@ multiple nodes concurrently.
 
 ### Windows and macOS
 
-Docker Buildx is included in [Docker Desktop](../desktop/index.md) for Windows
-and macOS.
-
-### Linux packages
-
-Docker Linux packages also include Docker Buildx when installed using the
-[DEB or RPM packages](../engine/install/index.md).
+Docker Buildx is included in [Docker Desktop](../desktop/index.md) for Windows, macOS, and Linux.
```

Comment:
Should we keep a separate heading for Linux installations (so a heading for "Docker Desktop on Windows, macOS, Linux", and one for Linux?)

Reply:
Have pushed another commit to fix this. Do my headings work?